include/mbedtls/check_config.h

```diff
@@ -763,14 +724,7 @@
 #if defined(MBEDTLS_SHA256_ALT) || defined(MBEDTLS_SHA256_PROCESS_ALT)
 #error "MBEDTLS_SHA256_*ALT can't be used with MBEDTLS_SHA256_USE_A64_CRYPTO_*"
 #endif
-#if defined(__aarch64__) && !defined(__ARM_FEATURE_CRYPTO)
-#error "Must use minimum -march=armv8-a+crypto for MBEDTLS_SHA256_USE_A64_CRYPTO_*"
-#endif
-#endif
 
-#if defined(MBEDTLS_SHA256_USE_A64_CRYPTO_ONLY) && \
```

Comment:
Again, this is the target CPU, not modifiers, so shouldn't it be required for both the application and the library?

Reply:
It is undefined at sha256.c line 58. It is not required for application.

Reply:
I have revert this change. It is not scope of this PR.